src/cljs/athens/views.cljs

```diff
@@ -86,7 +88,13 @@
 
          :else [:<>
                 (when @modal [filesystem/window])
-                [:div (stylefy/use-style app-wrapper-style)
+                [:div (use-style app-wrapper-style
+                                 {:class (join " "
+                                               [(cond
+                                                  (= (get-os) :windows) "os-windows"
+                                                  (= (get-os) :mac) "os-mac"
+                                                  (= (get-os) :linux) "os-linux")
+                                                (when (electron?) "is-electron")])})
```

Comment:
Could rewrite this map as

```
{:class [(case (get-os)
                                            :windows "os-windows"
                                            :mac "os-mac"
                                            :linux "os-linux")
                                          (when (electron?) "is-electron")]}
```

Only calls `(get-os)` once.

Don't need `join " "`, Reagent will automatically turn a list or vector of strings into space-delimited strings.